.gitignore

```diff
@@ -1,4 +1,5 @@
-
+/.vs
+/.vscode
```

Comment:
```suggestion
```
I feel like adding editor specific folders to the .gitignore is a bad rabbit hole, you end up adding them for every IDE. This belongs in a user level .gitignore instead.

Reply:
Yeah you are right. I'll change it then.